razorpay-payments.php

```diff
@@ -472,3 +425,8 @@ function woocommerce_add_razorpay_gateway($methods)
 
     add_filter('woocommerce_payment_gateways', 'woocommerce_add_razorpay_gateway' );
 }
+
+function razorpay_webhook_init()
+{
+    new RZP_Webhook();
+}
```

Comment:
1. newline at the end. 
2. I don't like doing everything in the constructor here